### PR TITLE
[HOPSWORKS-299] Add functionality for job unschedule

### PR DESCRIPTION
**Pull request by robzor92**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR meets the following requirements**
- [ ] Adds tests for the submitted changes (for bug fixes & features)
- [x] Passes the tests
- [x] HOPSWORKS JIRA issue has been opened for this PR
- [x] All commits have been squashed down to a single commit

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by robzor92** [comment]
https://github.com/logicalclocks/hops-testing/pull/199

File: hopsworks-api/src/main/java/io/hops/hopsworks/api/jobs/JobService.java [hunk 1]
```diff
@@ -1317,6 +1312,55 @@ public Response retryLogAggregation(@PathParam("appId") String appId,
...
+   * JSON object stating operation successful
+   * or not.
+   * <p>
+   * @param jobId
+   * @param sc
+   * @param req
+   * @return
+   */
+  @DELETE
+  @Path("/{jobId}/unschedule")
```
Comment (tkakantousis) [inline hopsworks-api/src/main/java/io/hops/hopsworks/api/jobs/JobService.java]: @robzor92 path should be `/{jobId}/schedule`. Delete action is applied on a resource, the uri should not end in an action. Actions should be in query params or in the payload.

Similarly in the v2 the updateSchedule path will change from 
```
@POST
@Path("/updateschedule/{jobId}")
```
to
```
@POST
@Path("/{jobId}/schedule")
```

Comment (robzor92) [reply hopsworks-api/src/main/java/io/hops/hopsworks/api/jobs/JobService.java]: Ok sure. Had a look in 510 and it is named jobId/*scheduled*. I will change the path and add queryparam.

Comment (tkakantousis) [reply hopsworks-api/src/main/java/io/hops/hopsworks/api/jobs/JobService.java]: It will be refactoring as part of HOPSWORKS-510

**Pull request merged**